api/api.go

```diff
@@ -62,6 +78,7 @@ func routes(host, secretKey string, router *mux.Router, dataStore store.DataStor
 	api.router.HandleFunc("/datasets/{id}/editions/{edition}/versions", api.getVersions).Methods("GET")
 	api.router.HandleFunc("/datasets/{id}/editions/{edition}/versions/{version}", api.getVersion).Methods("GET")
 	api.router.HandleFunc("/datasets/{id}/editions/{edition}/versions/{version}", api.privateAuth.Check(api.putVersion)).Methods("PUT")
+	api.router.HandleFunc("/datasets/{id}/editions/{edition}/versions/{version}/downloads", api.privateAuth.Check(api.PutVersionDownloads)).Methods("PUT")
```

Comment:
Typically a POST would go to the 'collection' endpoint, and a PUT would be made to a specific ID to update. Do the downloads have ID's we can PUT to? Probably worth double checking the API standards on this.

Reply:
As discussed this has been rewritten to use PUT on version